presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -144,6 +144,23 @@ String Functions
     Returns the starting position of the first instance of ``substring`` in
     ``string``. Positions start with ``1``. If not found, ``0`` is returned.
 
+.. function:: strpos(string, substring, instance) -> bigint
+
+    Returns the position of the n-th ``instance`` of ``substring`` in ``string``.
+    ``instance`` should be a positive number. Error is thrown if ``instance`` is ``0`` or negative.
```

Comment:
I would say `instance must be positive` and omit the following sentence.

Reply:
Done.